src/core/configuration/team-configuration-service.js

```diff
@@ -97,10 +101,24 @@ class TeamConfigurationService {
       .then(() => this.getAccountForAlias(teamConfiguration.getAlias(), teamId))
       .then(result => {
         if (!result) {
-          logger.error("tc:"+result+",alias" + teamConfiguration.getAlias()+",teamId="+teamId)
+          logger.error(
+            'tc:' +
+              result +
+              ',alias' +
+              teamConfiguration.getAlias() +
+              ',teamId=' +
+              teamId
+          );
```

Comment:
Can you use string literal here?
```suggestion
  logger.error(`tc: ${result}, alias ${teamConfiguration.getAlias()} teamId= ${teamId}`);
```

Reply:
I don't mind, just saying this change here is just because I did prettier.